homeassistant/components/deconz/config_flow.py

```diff
@@ -169,3 +169,25 @@ def __init__(self):
         user_input = {CONF_ALLOW_CLIP_SENSOR: True,
                       CONF_ALLOW_DECONZ_GROUPS: True}
         return await self.async_step_options(user_input=user_input)
+
+    @staticmethod
+    def async_get_options_flow():
+        """"""
+        return DeconzOptionsFlowHandler
```

Comment:
I think that the flow should be initialized when returned, that way we could create generic flows to be consumed by multiple integrations.

Reply:
Should options flow just expect to get the full entry? Right now config and options get passed in.